docs/ops/generation/RandomUniform_8.md

```diff
@@ -0,0 +1,195 @@
+## RandomUniform <a name="RandomUniform"></a> {#openvino_docs_ops_generation_RandomUniform_8}
+
+**Versioned name**: *RandomUniform-8*
+
+**Category**: Generation
+
+**Short description**: *RandomUniform* operation generates a sequence of random values from a uniform distribution.
+
+**Attributes**:
+
+* *output_type*
+
+    * **Description**: the type of the output.
+    * **Range of values**: "i32", "i64", "f16", "f32", "f64".
+    * **Type**: string
+    * **Default value**: "f32"
+    * **Required**: *Yes*
+
+* *seed*
+
+    * **Description**: global seed value.
+    * **Range of values**: positive integers
+    * **Type**: `int`
+    * **Default value**: None
+    * **Required**: *Yes*
+
+* *seed2*
+
+    * **Description**: operational seed value.
+    * **Range of values**: positive integers
+    * **Type**: `int`
+    * **Default value**: None
+    * **Required**: *Yes*
+
+**Inputs**:
+
+*   **1**: `shape` - 1D tensor of type *T_SHAPE* describing output shape. **Required.**
+
+*   **2**: `minval` - Scalar with type specified by the attribute *output_type*, defines the lower bound on the range of random values to generate (inclusive). Default value = 0.
+
+*   **3**: `maxval` - Scalar with type specified by the attribute *output_type*, defines the upper bound on the range of random values to generate (exclusive). Required for int64 and int32. Default value = 1.
+
+
+**Outputs**:
+
+* **1**: A tensor with type specified by the attribute *output_type* and shape defined by `shape` input tensor.
+
+**Types**
+
+* *T_SHAPE*: any supported integer type.
+
+
+**Detailed description**:
+
+*RandomUniform* operation generates random numbers from a uniform distribution in the range [*minval*, *maxval*). The generation algorithm is based on underlying random integer generator that uses Philox algorithm. Philox algorithm is counter based pseudo random generator, which produces unit32 values. Single invocation of Philox algorithm returns four result random values, depending on the given *key* and *counter* values. *Key* and *counter* are initialized with *seed* and *seed2* attributes respectively.
```

Comment:
Please, add the description of the initialization of *key* and *counter*.

Reply:
key and counter are initialized from attributes values, there are no extra calculations for them.

Reply:
Added an explicit description of initialization of key and counter.